### PR TITLE
[KYV-745] Changed the repository method count all account export datas by exported year...

### PR DESCRIPTION
**Pull request by martinldev**
Changed the repository method count all account export datas by exported year to work with LocalDate type - requires counting by date range from first to last day of the provided year. Local environment uses now a mock sftp server to mock the functionality of SAP in production.

### CODE REVIEW

File: orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingExportService.java [hunk 1]
```diff
@@ -55,17 +59,17 @@ public void exportAccountingData(AccountingExportDataDto exportData) {
...
 
         export(exportData.getXml(), filename);
 
         markAsExported(exportData);
     }
 
     public String constructAccountingExportFileName(String senderId, LocalDate exportDataTimestamp) {
-        int year = exportDataTimestamp.getYear();
-        int count = exportDataRepository.countAllByExportedStartsWith(Integer.toString(year));
+        LocalDate start = LocalDate.of(exportDataTimestamp.getYear(), 1, 1);
```
Comment (severikupari1) [inline orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingExportService.java:70]: Can you rename variable to be startOfYear

File: orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingExportService.java [hunk 2]
```diff
@@ -55,17 +59,17 @@ public void exportAccountingData(AccountingExportDataDto exportData) {
...
+        LocalDate end = LocalDate.of(exportDataTimestamp.getYear(), 12, 31);
```
Comment (severikupari1) [inline orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingExportService.java:71]: Could you change this variable to be endOfYear

**Review by severikupari1 — COMMENTED** [review]
Good job, can you rename those variables and then this is good to go. I checked and there was no good way to get min and Max day/mont of the year.

File: orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingExportService.java (hunk 1 above)
Comment (martinldev) [reply orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/AccountingExportService.java]: Sure, I just now pushed the fix for these variable names.

**Review by severikupari1 — APPROVED** [review]
Looks good.

**Pull request merged**